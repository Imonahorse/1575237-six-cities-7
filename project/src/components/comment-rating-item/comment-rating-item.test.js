import React from 'react';
import {render} from '@testing-library/react';
import CommentRatingList from '../comment-rating-list/comment-rating-list.jsx';

describe('Component: CommentRatingList', () => {
  it('should render CommentRatingList', () => {
    const inputChangeHandle = jest.fn();
    const fakeValue = '3';

    const {isFragment} = render(<CommentRatingList onInputChange={inputChangeHandle} isActive={fakeValue}/>);

    expect(isFragment).toMatchSnapshot();
  });
});
