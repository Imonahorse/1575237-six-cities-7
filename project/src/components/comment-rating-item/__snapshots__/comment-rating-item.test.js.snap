// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: CommentRatingList should render CommentRatingList 1`] = `undefined`;
